rotkehlchen/globaldb/schema.py

```diff
@@ -171,20 +220,44 @@
 );
 """
 
+DB_CREATE_CUSTOM_ASSET = """
+CREATE TABLE IF NOT EXISTS custom_assets(
+    identifier INTEGER NOT NULL PRIMARY KEY,
+    name TEXT NOT NULL,
+    symbol TEXT,
+    notes TEXT,
+    type TEXT
+);
+"""
+
+DB_CREATE_ASSET_COLLECTION_PROPERTIES = """
+CREATE TABLE IF NOT EXISTS asset_collection_properties(
```

Comment:
What is this? I don't see it used anywhere.

Reply:
This is the table that will collect `USDC` as symbol for the USDC token in all the chains when they are introduced in the frontend to avoid the `USD//C` case and similar. So is symbol + name + identifier of the token representing the rest